sympy/solvers/tests/test_solveset.py

```diff
@@ -1584,7 +1584,6 @@ def test_nonlinsolve_abs():
 def test_raise_exception_nonlinsolve():
     raises(IndexError, lambda: nonlinsolve([x**2 -1], []))
     raises(ValueError, lambda: nonlinsolve([x**2 -1]))
-    raises(NotImplementedError, lambda: nonlinsolve([(x+y)**2 - 9, x**2 - y**2 - 0.75], (x, y)))
```

Comment:
Rather than just removing this test it would be good to add a test showing what this actually returns now.

Reply:
I have now added the test in the `test_nonlinsolve_polysys` function.